### PR TITLE
Add logs-keeper to vagrant

### PR DESCRIPTION
**Pull request by sumo-drosiek**
###### Description

Logs keeper is script which keeps rotated files. It can be used to check if all logs were read and send correctly through the pipeline

---

###### Testing performed

- [ ] Redeploy fluentd and fluentd-events pods
- [ ] Confirm events, logs, and metrics are coming in

### CODE REVIEW

**Comment by sumo-drosiek** [comment]
Reopen of #1410 due to misclick and merge only partial changes

File: vagrant/logs-keeper/logs-keeper.py [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+│   └── [5774580]  sumologic
+│       └── [5774581]  5774571
+│           └── [5774571]  example_file.json -> /logs-keeper/docker/sumologic/5774571/example_file_in_containers.json
+└── [5774570]  docker
+    ├── [5774571]  example_file.json
+    └── [5774574]  sumologic
+        └── [5774578]  5774571
+            └── [5774571]  example_file_in_containers.json
+
+It has been created sumologic directory in both places (where the symlink is and where the link target is).
```
Comment (sumo-drosiek) [inline vagrant/logs-keeper/logs-keeper.py:35]:
```suggestion
It created sumologic directory in both places (where the symlink is and where the link target is).
```

File: vagrant/logs-keeper/logs-keeper.py [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+            └── [5774572]  example_file_in_containers.json
+
+Usage:
+
+KEEP_TIME=3600 ./logs-keeper <directory_path>
+"""
+
+logging.basicConfig(level=logging.INFO, format="%(asctime)s %(message)s")
+log = logging.getLogger()
+KEEP_TIME = int(os.getenv("KEEP_TIME_S", 120))
```
Comment (astencel-sumo) [inline vagrant/logs-keeper/logs-keeper.py:83]: Is it `KEEP_TIME` or `KEEP_TIME_S`? Rel. line 78

File: vagrant/logs-keeper/logs-keeper.py [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+    ├── [5774571]  example_file.json.1
+    └── [5774574]  sumologic
+        ├── [5774578]  5774571
+        |   └── [5774571]  example_file_in_containers.json
+        └── [5774573]  5774572
+            └── [5774572]  example_file_in_containers.json
+
+Usage:
+
+KEEP_TIME=3600 ./logs-keeper <directory_path>
```
Comment (sumo-drosiek) [inline vagrant/logs-keeper/logs-keeper.py:78]:
```suggestion
KEEP_TIME_S=3600 ./logs-keeper <directory_path>
```

File: vagrant/logs-keeper/logs-keeper.py (hunk 2 above)
Comment (sumo-drosiek) [reply vagrant/logs-keeper/logs-keeper.py:83]: `KEEP_TIME_S`

**Review by astencel-sumo — APPROVED** [review]

**Pull request merged**